### ISSUE TITLE
`experiment.bathymetry` seems to have a vertical coordinate (weird?) and does not have horizontal coordinates

### ISSUE THREAD

**Issue by navidcy** [COSIMA/regional-mom6#165]
Seems like the bathymetry has a vertical coordinate attached to it? See, e.g., plot title at [the example](https://regional-mom6.readthedocs.io/en/latest/demo_notebooks/reanalysis-forced.html#Check-out-your-domain:). Also, after running the example I noticed that the `experiment.bathymetry` data array does not include horizontal coordinates.

Should we drop the vertical coordinate and include the horizontal lat/lon coordinates? This way the plot will have proper axis with lat/lon instead of indices. Also, should we drop the vertical coordinate? Is it used anywhere or required by MOM6?

**Comment by ashjbarnes** [comment]
Yeah this is how MOM6 expects things. The nx and ny values correspond to those on the hgrid, so it reads in the lat/lon from there. We could add lat/lon for user experience though

**Comment by navidcy** [comment]
So, to clarify: MOM6 expect the bathymetry to have a vertical coordinate attached to it?
